static/js/message_edit.js

```diff
@@ -193,14 +193,15 @@ function edit_message(row, raw_content) {
     currently_editing_messages[message.id] = edit_obj;
     current_msg_list.show_edit_message(row, edit_obj);
 
-    initClipboard(row.find('.copy_message')[0]);
-
     form.keydown(_.partial(handle_edit_keydown, false));
 
     var message_edit_content = row.find('textarea.message_edit_content');
     var message_edit_topic = row.find('input.message_edit_topic');
     var message_edit_topic_propagate = row.find('select.message_edit_topic_propagate');
     var message_edit_countdown_timer = row.find('.message_edit_countdown_timer');
+    var copy_message = row.find('.copy_message');
+
+    initClipboard(copy_message[0]);
```

Comment:
Let's just put this three times in the three `if` statements (rather than adding and removing), since we already do that for `message_edit_content.prop('readonly', 'readonly')`.